app/models/manageiq/providers/ansible_tower/configuration_manager/refresher.rb

```diff
@@ -0,0 +1,27 @@
+module ManageIQ::Providers
+  module AnsibleTower
+    class ConfigurationManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
+      include ::EmsRefresh::Refreshers::EmsRefresherMixin
+
+      def parse_inventory(manager, _targets)
+        manager.with_provider_connection do |connection|
+          manager.api_version = connection.version
+          manager.save
```

Comment:
Is the api_version a requirement with this provider?  I'm only asking because I didn't see this step with the Foreman refresher anywhere - or do we always want to know the api version for reporting and pulling it here makes the most sense?